src/js/components/stock-movement/outbound/StockMovementOutboundTable.jsx

```diff
@@ -337,7 +337,7 @@ const StockMovementOutboundTable = ({
       width: 150,
       Cell: row => (<DateCell {...row} />),
     },
-  ], [requisitionStatuses]);
+  ], [requisitionStatuses, translate]);
```

Comment:
does it really rely on `translate`, when statuses are translated on the backend?

Reply:
I noticed that there was a problem with displaying status tooltips which are not translated on the backend and use the `translate` method on the frontend